packages/main/src/Popup.js

```diff
@@ -79,7 +79,15 @@ const metadata = {
 		/**
 		 * @private
 		 */
-		_disableInitialFocus: {
+		// _disableInitialFocus: {
```

Comment:
Not sure about this change as the _disableInitialFocus is kind of protected property that several components rely on

<img width="368" alt="Screenshot 2020-12-07 at 10 21 42 PM" src="https://user-images.githubusercontent.com/15702139/101401184-a1973780-38da-11eb-8253-8918854cb8b3.png">